auth/auth.go

```diff
@@ -0,0 +1,42 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/ONSdigital/go-ns/rchttp"
+	"github.com/ian-kent/go-log/log"
```

Comment:
This needs updating to be the go-ns log package

Reply:
Done 👍